### ISSUE TITLE
!exportToFile scoresheet marks the reader as the person who calls !exportToFile

### ISSUE THREAD

**Issue by Sam-per** [alopezlago/QuizBowlDiscordScoreTracker#58]
Games where the !exportToFile method is called by someone who isn't the reader (such as an admin) has both the reader and scorekeeper marked as the person who called the command.

**Steps to reproduce:**
1. Play an exportable game
2. Have an admin _who is not the reader_ call !exportToFile
3. The reader and scorekeeper will be listed as the admin

**Expected Behavior**
The reader is listed as the reader.

As an extension, do we want to have the scorekeeper marked as the person who calls !exportToFile?

**Comment by alopezlago** [comment]
Good catch; we should use lookup the reader from the game's `ReaderId` property and fill in their nickname, or their username if the nickname doesn't exist.

As for the scorekeeper field: the bot's really scorekeeping, but it felt weird putting the bot's name in the scorekeeper field. The reader seems like the closest scorekeeper after that. Do people have separate scorekeepers (who are admins) when using the bot?

**Comment by Sam-per** [comment]
> Do people have separate scorekeepers (who are admins) when using the bot?

From the use cases I've seen, there is not a separate scorekeeper when using the bot. There could be a case where a room is double staffed at a tournament, and the second staffer might be labeled as "scorekeeper" by the TD, even if the reader is the one keeping score and the bot is the one making the score sheet. I have not seen this case, but I'm also unsure if it can happen.

Granted, the bot may remove the need for a dedicated scorekeeper at tournaments, so maybe marking reader/scorekeeper both as the reader makes more sense.
